web/msg/send.go

```diff
@@ -26,12 +26,13 @@ func init() {
 //	  "text": "hi there"
 //	}
 type sendRequest struct {
-	OrgID       models.OrgID       `json:"org_id"       validate:"required"`
-	UserID      models.UserID      `json:"user_id"      validate:"required"`
-	ContactID   models.ContactID   `json:"contact_id"   validate:"required"`
-	Text        string             `json:"text"`
-	Attachments []utils.Attachment `json:"attachments"`
-	TicketID    models.TicketID    `json:"ticket_id"`
+	OrgID        models.OrgID       `json:"org_id"       validate:"required"`
+	UserID       models.UserID      `json:"user_id"      validate:"required"`
+	ContactID    models.ContactID   `json:"contact_id"   validate:"required"`
+	Text         string             `json:"text"`
+	Attachments  []utils.Attachment `json:"attachments"`
+	QuickReplies []string           `json:"quick_replies"`
```

Comment:
@norkans7 just in case we change the format of quick replies.. let's make this an object `{"text": "yes"}`

Reply:
same on the actual API endpoints

Reply:
@rowanseymour ?

Reply:
yes